src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -220,79 +228,166 @@ export const SelectRegistrationTypeField = () => {
             </IconButton>
           )}
         </Box>
-        <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', justifyContent: 'end', mb: '1rem' }}>
-          <FilterVintageIcon
-            color="primary"
-            titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-            fontSize="small"
+        <Paper
+          elevation={5}
+          sx={{
+            padding: '0.5rem',
+            bgcolor: 'secondary.main',
+            display: 'grid',
+            gap: '0.5rem',
+            gridTemplateAreas: {
+              xs: "'search-bar' 'publication-points-text'",
+              md: "'search-bar publication-points-text'",
+            },
+            gridTemplateColumns: { xs: '1fr', md: '1fr 1fr' },
+            mb: '1rem',
+          }}>
+          <TextField
+            data-testid={dataTestId.registrationWizard.resourceType.resourceTypeSearchField}
+            sx={{ bgcolor: 'white', maxWidth: '15rem', gridArea: 'search-bar' }}
+            placeholder={t('registration.resource_type.search_for_resource_type')}
+            type="search"
+            variant="outlined"
+            label={t('common.search')}
+            InputProps={{
+              startAdornment: <SearchIcon sx={{ mr: '1rem' }} />,
+            }}
+            onChange={(event) => {
+              setSearchValue(event.target.value);
+            }}
```

Comment:
sparer enda to linjer :D 
```suggestion
            onChange={(event) => setSearchValue(event.target.value)}
```